suse/module-setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+check() {
+	[ -n "$hostonly" -a -c /sys/class/infiniband_verbs/uverbs0 ] && return 0
+	[ -n "$hostonly" ] && return 255
+	return 0
+}
+
+depends() {
+	return 0
+}
+
+install() {
+	inst /etc/rdma/mlx4.conf
+	inst /etc/rdma/modules/infiniband.conf
+	inst /etc/rdma/modules/iwarp.conf
+	inst /etc/rdma/modules/opa.conf
+	inst /etc/rdma/modules/rdma.conf
+	inst /etc/rdma/modules/roce.conf
+	inst /usr/lib/mlx4-setup.sh
+	inst_multiple lspci setpci awk sleep
+	inst_rules 70-persistent-ipoib.rules 75-rdma-description.rules 90-rdma-hw-modules.rules 90-rdma-ulp-modules.rules
```

Comment:
What about persistent naming rule?

Reply:
Yup. Missed that one. Was working on a previous release and it didn't exist there.
Fixed in the new commit.
Seems like dracut is smart enough to package rdma_rename by itself (thus no inst command)